client/boot/project/wordpress-com.js

```diff
@@ -255,7 +239,7 @@ function reduxStoreReady( currentUser, reduxStore ) {
 	}
 
 	if ( config.isEnabled( 'keyboard-shortcuts' ) ) {
-		require( 'lib/keyboard-shortcuts/global' )( sites ); // GENERIC? WHY SITES
+		require( 'lib/keyboard-shortcuts/global' )( sitesFactory() );
```

Comment:
It turned out to be specific to WordPress.com and its urls. That's why it needs sites to know how to create urls having selected site. I'm wondering if that is used at all.